pkg/controller/plan/kubevirt.go

```diff
@@ -1379,7 +1458,20 @@ type VirtualMachine struct {
 // owner of the CDI DataVolume.
 func (r *VirtualMachine) Owner(dv *cdi.DataVolume) bool {
 	for _, vol := range r.Spec.Template.Spec.Volumes {
-		if vol.DataVolume.Name == dv.Name {
+		if vol.PersistentVolumeClaim.ClaimName == dv.Name {
+			return true
+		}
+	}
+
+	return false
+}
+
+//
+// Determine if `this` VirtualMachine is the
+// owner of the PersistentVolumeClaim.
+func (r *VirtualMachine) OwnerPVC(pvc *core.PersistentVolumeClaim) bool {
+	for _, vol := range r.Spec.Template.Spec.Volumes {
+		if vol.PersistentVolumeClaim.ClaimName == pvc.Name {
```

Comment:
how come that we could have a PVC that would not be owned by the VM?

Reply:
you mean that before we add/replace the owner of the PVC, it's owned only by the DV and this method is used to check the correlation with the VM? can it be replaced with checking labels - after all, we know for which VM we create the PVCs..

Reply:
> This method actually not being used.

even better - then let's drop it